app/controllers/homes_controller.rb

```diff
@@ -1,7 +1,28 @@
 class HomesController < ApplicationController
   def index
-    @pagy, @products = pagy Product.newest, items: Settings.items_per_page
+    @products = Product.newest
+    @products = Product.search_by_name params[:name] if params[:name].present?
```

Comment:
gộp dòng 3, 4 nè em

Reply:
dòng 3 em lấy toàn bộ sản phẩm, dòng 4 em tìm theo tên 2 chức năng khác nhau thì gộp ntn ạ ?